src/webui/www/public/scripts/dynamicTable.js

```diff
@@ -775,22 +776,22 @@ var TorrentsTable = new Class({
             this.columns['state_icon'].updateTd = function (td, row) {
                 var state = this.getRowValue(row);
 
-                if (state == "forcedDL" || state == "metaDL")
+                if (state === "forcedDL" || state === "metaDL")
```

Comment:
It's not required, but I think the question should be the other way around. You should use `===` whenever possible, and only use `==` when required. In this case, state it always a string, so `===` can be used.